rust/sealevel/environments/mainnet3/warp-routes/SOLX-nitro/token-config.json

```diff
@@ -9,6 +9,16 @@
     "interchainGasPaymaster": "AkeHBbE5JkwVppujCQQ6WuxsVsJtruBAjUo6fDCFp6fF",
     "owner": "BNGDJ1h9brgt6FFVd8No1TVAH48Fp44d7jkuydr1URwJ"
   },
+  "solaxy": {
+    "type": "synthetic",
+    "decimals": 6,
+    "remoteDecimals": 18,
+    "name": "Solaxy",
+    "symbol": "SOLX",
+    "uri": "https://raw.githubusercontent.com/hyperlane-xyz/hyperlane-registry/f50f80c269077c9ff184b1ba22d1c72618bbd80a/deployments/warp_routes/SOLX/metadata.json",
+    "interchainGasPaymaster": "3SDcBbfZUF9ykR3yudYN198ja3KJSq17Q3rKw7JKfJR6",
```

Comment:
I think this should be `7z1QjgLLxowA9RA9QBuiC72Tc6aC1XjtA5zdH9LJHm1a` instead (the overhead IGP account)